src/lucky_web/log_handler.cr

```diff
@@ -12,12 +16,18 @@ class LuckyWeb::LogHandler
     elapsed = Time.now - time
     elapsed_text = elapsed_text(elapsed)
 
-    @io.puts "#{context.request.method} #{context.response.status_code} #{context.request.resource.colorize(:green)} (#{elapsed_text})"
+    timestamp = if settings.show_timestamps
+                  Time::Format::ISO_8601_DATE_TIME.format(time)
+                else
+                  ""
+                end
+
+    @io.puts "#{context.request.method} #{context.response.status_code} #{context.request.resource.colorize(:green)} #{timestamp} (#{elapsed_text})"
```

Comment:
I think this is almost just right, but there will be two spaces if no timestamp is show. I'm thinking you could add the space when formatting the time: `" #{Time::Format::ISO_8601_DATE_TIME.format(Time.now}"`

Reply:
Yes, you are right